From 6968966d5f8506c14626cbd1fcfaafcbd68593f5 Mon Sep 17 00:00:00 2001
Message-Id: <6968966d5f8506c14626cbd1fcfaafcbd68593f5.1352202113.git.erack@redhat.com>
From: Markus Mohrhard <markus.mohrhard@googlemail.com>
Date: Thu, 11 Oct 2012 14:32:49 +0200
Subject: [PATCH] kill this hard coded sheet limit for xlsx import,
 rhbz#864858
MIME-Version: 1.0
Content-Type: multipart/mixed; boundary="------------erAck-patch-parts"

This is a multi-part message in MIME format.
--------------erAck-patch-parts
Content-Type: text/plain; charset=UTF-8; format=fixed
Content-Transfer-Encoding: 8bit


(cherry picked from commit 458d68fffa883bc706638299e5a5b2d1399beb0d)

Conflicts:

	sc/source/filter/oox/addressconverter.cxx

Backported change of sc/source/filter/oox/addressconverter.cxx to
oox/source/xls/addressconverter.cxx, it's just another hard coded value
here in 3-5, but ...

Change-Id: I8fe6523afb11e5b24b8c83784e1198e8c694922a
Signed-off-by: Eike Rathke <erack@redhat.com>
---
 oox/source/xls/addressconverter.cxx |    3 ++-
 1 files changed, 2 insertions(+), 1 deletions(-)


--------------erAck-patch-parts
Content-Type: text/x-patch; name="0001-kill-this-hard-coded-sheet-limit-for-xlsx-import-rhb.patch"
Content-Transfer-Encoding: 8bit
Content-Disposition: attachment; filename="0001-kill-this-hard-coded-sheet-limit-for-xlsx-import-rhb.patch"

diff --git a/oox/source/xls/addressconverter.cxx b/oox/source/xls/addressconverter.cxx
index fd7217a..d6e2a38 100644
--- a/oox/source/xls/addressconverter.cxx
+++ b/oox/source/xls/addressconverter.cxx
@@ -58,7 +58,8 @@ using ::rtl::OUStringToOString;
 namespace {
 
 //! TODO: this limit may change, is there a way to obtain it via API?
-const sal_Int16 API_MAXTAB          = 255;
+//! same value as MAXTAB in sc/inc/address.hxx
+const sal_Int16 API_MAXTAB          = 9999;
 
 const sal_Int32 OOX_MAXCOL          = static_cast< sal_Int32 >( (1 << 14) - 1 );
 const sal_Int32 OOX_MAXROW          = static_cast< sal_Int32 >( (1 << 20) - 1 );

--------------erAck-patch-parts--


